package.json

```diff
@@ -2,22 +2,24 @@
   "name": "eslint-find-rules",
   "version": "0.0.0-semantically-released",
   "description": "Find built-in ESLint rules you don't have in your custom config.",
-  "main": "src/lib/rule-finder.js",
+  "main": "dist/lib/rule-finder.js",
   "scripts": {
     "cover": "nyc --reporter=lcov --reporter=text npm test",
+    "lint": "xo",
     "test": "mocha --recursive",
-    "lint": "eslint --ignore-pattern test/fixtures .",
     "update-contributors": "all-contributors generate",
     "commit": "git-cz",
     "validate": "npm-run-all --parallel lint cover --sequential check-coverage",
     "check-coverage": "nyc check-coverage --statements 100 --branches 100 --functions 100 --lines 100",
+    "build": "babel src -d dist --presets es2015",
     "report-coverage": "cat ./coverage/lcov.info | node_modules/.bin/codecov",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "travis-after-all": "travis-after-all && npm run report-coverage && npm run semantic-release"
   },
+  "files": ["dist"],
```

Comment:
Yet, nobody does it.

You're correct about the dependency issue, but that's why you can use the related rule in eslint-plugin-import to correct that.

It's certainly a fine argument that "files" could be made *safe* - but it's unarguably unsafe at the moment. Similarly, however, any tool that makes "files" safe could also ensure "npmignore" doesn't omit anything. I'd be very interested in evaluating a tool that covered both cases.

The only reasonable choice for now is the thing that's safe by default - which is npmignore. The "consequence" of "slightly higher filesize" is a complete non-problem, one that's not worth solving at the cost of safety.